### PR TITLE
Add default metadata as module; and two work types: default_work, etd

### PR DESCRIPTION
**Pull request by zhang4952**
This branch is for DSpace migration and should be merged to Hyrax

### CODE REVIEW

File: app/models/concerns/scholars_archive/default_metadata.rb [hunk 1]
```diff
@@ -4,8 +4,57 @@ module DefaultMetadata
...
+
+	  property :date_available, predicate: ::RDF::URI.new("http://purl.org/dc/terms/description") do |index|
+	    index.as :stored_searchable
+	  end
+
+	  property :date_updated, predicate: ::RDF::URI.new("http://purl.org/dc/terms/date") do |index|
+	    index.as :stored_searchable
+	  end
+
+	  property :citation, predicate: ::RDF::URI.new("http://purl.org/dc/terms/identifier") do |index|
```
Comment (wickr) [inline app/models/concerns/scholars_archive/default_metadata.rb]: This isn't right for Citation.

File: app/models/concerns/scholars_archive/default_metadata.rb [hunk 2]
```diff
@@ -4,8 +4,57 @@ module DefaultMetadata
...
+	    index.as :stored_searchable
+	  end
+
+	  property :isbn, predicate: ::RDF::URI.new("http://purl.org/dc/terms/identifier") do |index|
+	    index.as :stored_searchable
+	  end
+
+	  property :identifier_uri, predicate: ::RDF::URI.new("http://purl.org/dc/terms/identifier") do |index|
```
Comment (wickr) [inline app/models/concerns/scholars_archive/default_metadata.rb]: We can't have multiple properties with the same predicate.

File: app/models/concerns/scholars_archive/default_metadata.rb (hunk 1 above)
Comment (zhang4952) [inline app/models/concerns/scholars_archive/default_metadata.rb]: citation is a kind of identifier now, as dc.identifier.citation

File: app/models/concerns/scholars_archive/default_metadata.rb (hunk 2 above)
Comment (zhang4952) [inline app/models/concerns/scholars_archive/default_metadata.rb]: how about identifieruri, or uri_identifier?

**Pull request merged**